modules/mbedtls/crypto_mbedtls.cpp

```diff
@@ -384,3 +451,12 @@ Vector<uint8_t> CryptoMbedTLS::decrypt(Ref<CryptoKey> p_key, Vector<uint8_t> p_c
 	copymem(out.ptrw(), buf, size);
 	return out;
 }
+
+PackedByteArray CryptoMbedTLS::hmac_digest(HashingContext::HashType p_hash_type, PackedByteArray p_key, PackedByteArray p_msg) {
```

Comment:
Move this to `core/crypto.h` use `Ref<HMACContext>(HMACContext::create())` instead of referencing `HMACContextMbedTLS` directly.
And make sure that the resulting reference is valid before calling functions on it:
```cpp
Ref<HMACContext> ctx = Ref<HMACContext>(HMACContext::create());
ERR_FAIL_COND_V(ctx.is_null(), PackedByteArray());
Error err = ctx->start(p_hash_type, p_key);
ERR_FAIL_COND_V(err != OK, PackedByteArray());
err = ctx->update(p_msg);
ERR_FAIL_COND_V(err != OK, PackedByteArray());
return ctx->finish();
```